spaceR/spaceR/Controllers/RocketsControlller.swift

```diff
@@ -11,30 +11,28 @@ import UIKit
 
 class RocketsController: UIViewController {
     
-    @IBOutlet weak var tableView: UITableView!
-    
-    @IBOutlet weak var emailIdLabel: UILabel!
-    var emailText = ""
-    private var rockets: [Rocket] = []
+    @IBOutlet weak var rocketTableView: UITableView!
```

Comment:
This can be private 😄

Reply:
Okay will do

Reply:
This still needs to be addressed 😄

Reply:
Cool. I actually removed this for now because i don't need it anymore